app/views/admin/auctions/_time_selector.html.erb

```diff
@@ -1,11 +1,11 @@
 <div>
-<select class="time-input" name="auction[<%=field%>_at(1i)]">
+<select class="time-input" name="auction[<%=field%>_at(1i)]" id="auction_<%=field%>_at_1i">
```

Comment:
That wasn't working for me for some reason. Dunno. Anyhow, I think the persistence issue was loading with no timezone + AM/PM and other time saving issues